tests/bluetooth/tester/src/btp_bas.c

```diff
@@ -0,0 +1,431 @@
+/*
+ * Copyright (c) 2024 Demant A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stddef.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/services/bas.h>
+#include "btp/btp.h"
+#include "btp/btp_bas.h"
+
+static uint8_t bas_supported_commands(const void *cmd, uint16_t cmd_len, void *rsp,
+				      uint16_t *rsp_len)
+{
+	struct btp_bas_read_supported_commands_rp *rp = rsp;
+
+	/* octet 0 */
+	tester_set_bit(rp->data, BTP_BAS_READ_SUPPORTED_COMMANDS);
+	tester_set_bit(rp->data, BTP_BAS_SET_BATTERY_LEVEL);
+	tester_set_bit(rp->data, BTP_BAS_BLS_SET_BATTERY_PRESENT);
```

Comment:
some of those handlers are conditional (under ifdefs) so this should also reflect that  (autopts doesn't use it yet, but eventually we want to start relying on supported commands)